src/ui/TopBanner/TopBanner.tsx

```diff
@@ -115,19 +119,19 @@ const IconSymbol: React.FC = () => {
 
 interface TopBannerProps {
   variant?: Variants
-  localStorageKey: string
+  localStorageKey?: string
 }
 
 const TopBannerRoot: React.FC<React.PropsWithChildren<TopBannerProps>> = ({
   variant = 'default',
-  localStorageKey,
+  localStorageKey = null,
```

Comment:
This default value of `null` conflicts with the types defined in the props interface. Adding in `?` makes the field either it's value in this case a `string` or `undefined`, so you can just skip assigning a default value here.